cmd/razorpay-mcp-server/sse.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	stdlog "log"
+	"log/slog"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	rzpsdk "github.com/razorpay/razorpay-go"
+
+	"github.com/razorpay/razorpay-mcp-server/pkg/log"
+	"github.com/razorpay/razorpay-mcp-server/pkg/mcpgo"
+	"github.com/razorpay/razorpay-mcp-server/pkg/razorpay"
+)
+
+// sseCmd starts the mcp server in sse transport mode
+var sseCmd = &cobra.Command{
+	Use:   "sse",
+	Short: "start the sse server",
+	Run: func(cmd *cobra.Command, args []string) {
+		logPath := viper.GetString("log_file")
+		log, close, err := log.New(logPath)
+		if err != nil {
+			stdlog.Fatalf("create logger: %v", err)
+		}
+		defer close()
+		client := rzpsdk.NewClient("", "")
```

Comment:
lets make client every time in every request - go SDK does not do conn pool at present.